statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -2894,6 +2894,76 @@ def zvalues(self):
         """
         return self.params / self.bse
 
+    def get_rsquared(self, baseline="rwdrift"):
+        if self.model.k_endog > 1:
+            raise NotImplementedError(
+                "To be Implemented for Multivariate Model")
+        if baseline == "rwdrift":
+            return self.rsquared_rwdrift
+        elif baseline == "mean":
+            return self.rsquared_mean
+        elif baseline == "seasonal":
+            raise self.rsquared_seasonal
+        else:
+            raise NotImplementedError(
+                "baseline must be in ['rwdrife', 'mean', 'seasonal']")
+
+    @cache_readonly
+    def sse_finite(self):
```

Comment:
Let's move this to be a local variable within `get_rsquared` for now, rather than a property.

Reply:
i am pulling all calc under get_rsquared(). All rsquared property do nothing but will just call get_rsquared()